hugegraph-common/pom.xml

```diff
@@ -11,7 +27,7 @@
     </parent>
 
     <artifactId>hugegraph-common</artifactId>
-    <name>${artifactId}</name>
+    <name>${project.artifactId}</name>
```

Comment:
Separate from this PR?  since this pr can be used as a guide for other repos.

Reply:
separated it already, although other repos should also update it